datasets/logiqa_en/README.md

```diff
@@ -0,0 +1,151 @@
+---
+YAML tags:
+- copy-paste the tags obtained with the tagging app: https://github.com/huggingface/datasets-tagging
```

Comment:
Missing the tags here. As indicated, you can generate them using the tagging app available here: https://github.com/huggingface/datasets-tagging

Reply:
Done!